platform/src/main/java/org/stellar/anchor/platform/data/JdbcSep24RefundPayment.java

```diff
@@ -0,0 +1,28 @@
+package org.stellar.anchor.platform.data;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import javax.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import org.stellar.anchor.sep24.Sep24RefundPayment;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "sep24_refund_payment")
+public class JdbcSep24RefundPayment implements Sep24RefundPayment {
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  @Column(name = "sep24_refund_payment_id")
+  Long jdbcId;
```

Comment:
💡  What about using UUID instead of auto-incremented Long?
According to [baeldung](https://www.baeldung.com/hibernate-identifiers#1-auto-generation) you can simply use UUID instead of long and it will work automatically:

```suggestion
  UUID jdbcId;
```

Reply:
Changed from Long to UUID..